### PR TITLE
Changed a URL to play the game

### PR DESCRIPTION
**Pull request by voltel**
New URL is https://voltel.github.io/reviewing-a-pull-request/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ## Welcome to awesome game of Tetris
 
 A project based learning activity for people who are getting started with Git and GitHub.
-You can play the game at: https://voltel.github.io/github-games/
+You can play the game at: https://voltel.github.io/reviewing-a-pull-request/
```
Comment (voltel) [inline README.md:4]: Are you sure?

Comment (voltel) [reply README.md:4]: Yes, I am sure

**Review by robtel — APPROVED** [review]
I generally approve.

**Review by robtel — APPROVED** [review]
Ok. Approved

**Review by robtel — APPROVED** [review]
Yes

**Comment by robtel** [comment]
Ok.

File: README.md (hunk 1 above)
Comment (robtel) [reply README.md:4]: That is fine.

**Review by robtel — APPROVED** [review]

**Pull request merged**